packages/pyright-internal/src/parser/tokenizer.ts

```diff
@@ -1655,41 +1660,35 @@ export class Tokenizer {
                     flags |= StringTokenFlags.ReplacementFieldStart;
                     break;
                 } else {
-                    escapedValueParts.push(this._cs.currentChar);
+                    escapedValueLength++;
                     this._cs.moveNext();
-                    escapedValueParts.push(this._cs.currentChar);
+                    escapedValueLength++;
                     this._cs.moveNext();
                 }
             } else if (isInNamedUnicodeEscape && this._cs.currentChar === Char.CloseBrace) {
                 isInNamedUnicodeEscape = false;
-                escapedValueParts.push(this._cs.currentChar);
+                escapedValueLength++;
                 this._cs.moveNext();
             } else if (isFString && this._cs.currentChar === Char.CloseBrace) {
                 if (inFormatSpecifier || this._cs.nextChar !== Char.CloseBrace) {
                     flags |= StringTokenFlags.ReplacementFieldEnd;
                     break;
                 } else {
-                    escapedValueParts.push(this._cs.currentChar);
+                    escapedValueLength++;
                     this._cs.moveNext();
-                    escapedValueParts.push(this._cs.currentChar);
+                    escapedValueLength++;
                     this._cs.moveNext();
                 }
             } else {
-                escapedValueParts.push(this._cs.currentChar);
+                escapedValueLength++;
                 this._cs.moveNext();
             }
         }
 
-        // String.fromCharCode.apply crashes (stack overflow) if passed an array
-        // that is too long. Cut off the extra characters in this case to avoid
-        // the crash. It's unlikely that the full string value will be used as
-        // a string literal or a docstring, so this should be fine.
-        if (escapedValueParts.length > maxStringTokenLength) {
-            escapedValueParts = escapedValueParts.slice(0, maxStringTokenLength);
-            flags |= StringTokenFlags.ExceedsMaxSize;
```

Comment:
Are you removing the generation of ExceedsMaxSize? If this is no longer a limitation, let's completely remove ExceedsMaxSize from the StringTokenFlags enumeration along with any references to it.

Reply:
Thanks, will do.